### PR TITLE
docs: Small typo in urls.rst

### PR DESCRIPTION
**Pull request by obozdag**
**Description**
Small typo in urls.rst

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/general/urls.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ by different servers, but we will show examples for the two most common web serv
 Apache Web Server
 -----------------
 
-Apache must have the *mod_rewrite* extension enabled. If it does, you can use a ``.htaccess`` file with some simple rules.
+Apache must have the *mod_rewrite* extension enabled. If it has, you can use a ``.htaccess`` file with some simple rules.
```
Comment (kenjis) [inline user_guide_src/source/general/urls.rst:78]: Is this a typo? I don't know.

Comment (datamweb) [reply user_guide_src/source/general/urls.rst:78]: The second sentence is grammatically correct: "Apache must have the mod_rewrite extension enabled. If it has, you can use a .htaccess file with some simple rules."

Here's the breakdown:

"Apache must have the mod_rewrite extension enabled." This is the main statement, indicating a requirement.

"If it has," is a conditional clause beginning with "if," indicating that the action in the main clause ("you can use a .htaccess file") is dependent on the condition stated in the if-clause ("Apache has the mod_rewrite extension enabled").

This construction is grammatically sound and clearly conveys the condition for using the .htaccess file.

**Comment by obozdag** [comment]
@kenjis 
Apache has the mod_rewrite extension enabled. If it has....
Apache mod_rewrite extension is enabled. If it is...
Apache enables the mod_rewrite extension. If it does...
Apache must have the mod_rewrite extension enabled. If it has the mod_rewrite extension enabled, ...
Apache must have the mod_rewrite extension enabled. If it does have the mod_rewrite extension enabled...

I am not a native English speaker but after your question I tried to check if it is a typo or not. Possibly we need expert help. I am not 100 percent sure.

**Comment by kenjis** [comment]
If a native English speaker thinks this is better and approves, it will merge this.

**Review by paulbalandan — COMMENTED** [review]
I believe both cases are correct. We can use fillers to convey the meaning better.

Old: If it does [have the "mod_rewrite" extension],
New: If it has [the "mod_rewrite" extension],

**Review by paulbalandan — APPROVED** [review]

**Pull request closed without merging**